### PR TITLE
More short closures

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->beforeNormalization()
-                ->ifTrue(function ($v) { return !isset($v['assets']) && isset($v['templating']) && class_exists(Package::class); })
+                ->ifTrue(fn ($v) => !isset($v['assets']) && isset($v['templating']) && class_exists(Package::class))
```
Comment (GromNaN) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php:77]: Most of this closures can be `static`. They were in your [previous PR](https://github.com/symfony/symfony/pull/51121/files#diff-0ef45febdaaf1b66bf33827186af3cafaee167ba59c902b237c8492855bab73eL71) on short closures.

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php:77]: less visual debt FTW, making them static wouldn't provide anything

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
fabbot has something for you and tests seem broken because of this PR :)

**Comment by nicolas-grekas** [comment]
Oops, should be good now.

**Pull request merged**